$NetBSD: patch-ac,v 1.1 2008/01/06 01:07:05 heinz Exp $

  Status: new (web interface for bug reports on platon.sk did not work)
  Only change the prefix if it starts with a dot.

--- configure.in.orig	2004-02-17 17:37:47.000000000 +0100
+++ configure.in
@@ -60,7 +60,10 @@ if test "X$prefix" != "XNONE"; then
 		mkdir -p "$prefix";
 		chmod 755 "$prefix";
 	fi
-	prefix="`cd $prefix && pwd`"
+	case	"X$prefix" in
+		X.*)	prefix="`cd $prefix && pwd`"
+			;;
+	esac
 fi
 
 AC_SUBST(PACKAGE)
